From: Phil Elwell <phil@raspberrypi.com>
Date: Thu, 11 Jun 2020 11:22:38 +0100
Subject: ARM: dts: Set brcm,enable-l1ss for CM4

Enable the PCIE L1SS on Compute Module 4. It's possible that this is
also the right thing to do for Pi 4, but it has been working as is
up to now.

Signed-off-by: Phil Elwell <phil@raspberrypi.com>
---
 arch/arm/boot/dts/bcm2711-rpi-cm4.dts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm/boot/dts/bcm2711-rpi-cm4.dts b/arch/arm/boot/dts/bcm2711-rpi-cm4.dts
index f616d3f23f14..2ea54b92fbca 100644
--- a/arch/arm/boot/dts/bcm2711-rpi-cm4.dts
+++ b/arch/arm/boot/dts/bcm2711-rpi-cm4.dts
@@ -252,6 +252,10 @@
 	/delete-node/ wifi-pwrseq;
 };
 
+&pcie0 {
+       brcm,enable-l1ss;
+};
+
 &mmcnr {
 	pinctrl-names = "default";
 	pinctrl-0 = <&sdio_pins>;
